types/formidable/tslint.json

```diff
@@ -1,19 +1,6 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "array-type": false,
-        "ban-types": false,
-        "dt-header": false,
-        "no-consecutive-blank-lines": false,
-        "no-var-keyword": false,
-        "object-literal-shorthand": false,
-        "only-arrow-functions": false,
-        "prefer-const": false,
-        "space-before-function-paren": false,
-        "strict-export-declare-modifiers": false,
-        "trim-file": false,
-        "triple-equals": false,
-        "whitespace": false,
         "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
```

Comment:
why this one is needed? (the npm package name is now the same as one published + repository is the same)

Reply:
there was an issue before. I will remove that.